improve module name readability by making Module/Directory header not to wrap

--- ./templates/en/checkins.tpl.html~	2005-03-30 11:53:45.000000000 +0300
+++ ./templates/en/checkins.tpl.html	2005-04-04 23:45:15.000000000 +0300
@@ -48,7 +48,7 @@
                 </td>
                 <td class="default_white" nowrap>Date</td>
                 <td class="default_white">User</td>
-                <td class="default_white">Module / Directory</td>
+                <td class="default_white" nowrap>Module / Directory</td>
                 <td class="default_white" nowrap>File</td>
                 <td width="55%" class="default_white">Commit Message</td>
               </tr>
